libs/core/src/components/pds-accordion/pds-accordion.scss

```diff
@@ -23,7 +23,7 @@ details {
   --spacing-summary-padding-inline-start: var(--pine-spacing-100);
   --spacing-summary-padding-inline-end: var(--pine-spacing-050);
 
-  --typography-default: var(--pine-font-weight-medium) var(--pine-font-size-100)/var(--pine-line-height-150) var(--pine-font-family-inter);
+  --typography-default: var(--pine-typography-body-md-medium);
```

Comment:
📓 For later discussion

Could we rename this? This seems redundant.

--[DS name]-[category]-[target]-[font-weight]. I can explain at a later time